main/zoe/api/zoe.md

```diff
@@ -3,217 +3,204 @@
+
+The `issuerKeywordRecord` is a record mapping string names (keywords) 
+to `issuers`, such as `{ Asset: quatlooIssuer}`. Keywords must begin 
+with a capital letter and must be ASCII. Parties to the contract will 
+use the keywords to index their proposal and their payments. 
+
+`terms` are values used by this contract instance, such as the 
+number of bids an auction will wait for before closing. These values may
+be different for different instances of the same contract, but the contract
+defines what variables need their values passed in as `terms`. 
+
+It returns a `promise` for a `StartInstanceResult` object. The object consists of:
+- `creatorFacet` `{any}`
+- `publicFacet` `{any}`
+- `instance` `{Instance}`
+- `creatorInvitation` `{Payment | undefined}`
+
+A `publicFacet` and a `creatorFacet` are objects available via Zoe to anyone knowing 
+the `instance` they are associated with. The `publicFacet` is used for general queries 
+and actions, such as getting a current price or creating public invitations. Since a 
+facet is defined just as any other object, you add methods to them just like you would any object.
+
+`creatorInvitation` is an invitation that the contract instance creator can use. 
+It is usually used in contracts where the creator immediately sells 
+something (auctions, swaps, etc.), so it's helpful for the creator to have 
+an invitation to escrow and sell goods.
 
 ```js
-  const offerNotifer = E(zoe).getOfferNotifier(offerHandle);
-  const { value, updateHandle, done } = offerNotifier.getUpdateSince();
-  if (done) {
-   <drop offer from list>
-  }
-  newValue = value;
-  waitForNextUpdate(offerNotifier, updateHandle);
+const issuerKeywordRecord = { 
+  'Asset' : moolaIssuer, 
+  'Price' : quatlooIssuer 
+};
+const terms = { numBids: 3 };
+const { invite, instanceRecord } = await E(zoe).startInstance(
+  creatorFacet, publicFacet, creatorInvitation);
 ```
+## E(Zoe).offer(invitation, proposal, paymentKeywordRecord)
+- `invitation` `{Invitation|Promise<Invitation>}`
+- `proposal` `{Proposal}`
+- `paymentKeywordRecord` `{PaymentKeywordRecord}`
+- Returns: `{Promise<UserSeat>}`
+
+Used to redeem the `invitation` provided as the first argument.
+
+To redeem an `invitation`, a user normally provides a `proposal` (their
+rules for an offer) as well as `payments` to be escrowed by Zoe.  If
+either the `proposal `or `payments` are empty, indicate this by
+omitting that argument or passing `undefined`, instead of passing an
+empty record.
+
+The `proposal` has three parts: 
+- `want`: An object with keywords as keys and `amounts` as values.
+- `give`: An object with keywords as keys and `amounts` as values.
+- `exit`: Specifies the payout-liveness policy Zoe can guarantee for the offer.
+
+`paymentKeywordRecord` is a record with keywords as keys, with
+ values of the actual `payments` to be escrowed. A `payment` is
+ expected for every rule under `give`.
+ 
+ `offer()` returns a `promise` for a `userSeat`. 
```

Comment:
we need a description of the API for userSeat.

Reply:
It's coming shortly. After this lands, I'll do a hopefully quick v2 that incorporates both objects like UserSeat and ZCFSeat  and the current contents of the Records doc page (updated to Alpha).